### PR TITLE
Prettier and adjust code for formatter

### PR DESCRIPTION
**Pull request by sergey-shandar**
Prettier hates functional programming...

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -133,14 +133,17 @@ const parseRef = (ref: string): Ref => {
   return i < 0 ? { url: ref, pointer: '' } : { url: ref.substr(0, i), pointer: ref.substr(i + 1) }
 }
 
-const getRefs = (j: json.Json): it.IterableEx<string> =>
-  json.isObject(j)
-    ? stringMap
+const getRefs = (j: json.Json): it.IterableEx<string> => {
+  if (json.isObject(j)) {
```
Comment (sergey-shandar) [inline src/index.ts]: 👎

File: src/dev-ops.ts [hunk 2]
```diff
@@ -57,14 +57,14 @@ const parseGitFileChangeKind = (line: string) => {
  * Currently, the algorithm is recognizing Azure Dev Ops Pull Request if the `env` has
  * `SYSTEM_PULLREQUEST_TARGETBRANCH`. `cwd` should point to the source Git repository.
  */
-export const createPullRequestProperties = async ({ cwd, env }: cli.Config)
-  : Promise<PullRequestProperties | undefined> => {
-
-  const targetBranch = env.SYSTEM_PULLREQUEST_TARGETBRANCH
+export const createPullRequestProperties = async (
+  config: cli.Config
```
Comment (sergey-shandar) [inline src/dev-ops.ts]: Prettier forced me to use this syntax instead of destruction 👎

File: src/git.ts [hunk 3]
```diff
@@ -4,27 +4,25 @@
...
+  | { readonly remote: readonly ['add', string, string] }
+  | { readonly clone: readonly [string, string] }
+  | { readonly diff: readonly ['--name-status' | '--name-only', '--no-renames', string, string] }
 
-export const repository = (repositoryPath: string) =>
-  async (command: Command) => {
+export type RunCommand = (command: Command) => Promise<childProcess.ExecResult>
+
+export const repository = (repositoryPath: string): RunCommand => {
+  return async (command: Command) => {
```
Comment (sergey-shandar) [inline src/git.ts]: Prettier forced me to use `{}` instead of cool lambda expressions.... Ahhhh… 👎

File: .prettierrc [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "semi": false,
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "es5"
```
Comment (timotheeguerin) [inline .prettierrc]: change to "all"

File: .prettierrc [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "semi": false,
+  "printWidth": 100,
```
Comment (vladbarosan) [inline .prettierrc]: 120

**Pull request merged**